### ISSUE TITLE
Where is the attention_mask or causal_mask?

### ISSUE THREAD

**Issue by guotong1988** [google-research/pegasus#59]
According to https://sshleifer.github.io/blog_v2/jupyter/2020/03/12/bart.html
It seems there is no mask in https://github.com/google-research/pegasus/blob/master/pegasus/models/transformer.py
Thank you very much.

**Comment by guotong1988** [comment]
attention.upper_triangle_bias
